### PR TITLE
Modified makefile and changed dir structure

### PR DESCRIPTION
**Pull request by vclagudu**
Changed the directory structure in a way that is supported by the makefile. Also changed the makefile targets a bit so that there was no redundancy, after adding the 270 FPGA targets.

### CODE REVIEW

**Review by HtetWYan — APPROVED** [review]
Looks good. Htet Approved!

**Review by StarsDown64 — CHANGES_REQUESTED** [review]
@lagudu-glitch Can we restore "insert_project_name_here.sv" so that we can have both the FPGA top and ASIC top?

**Comment by StarsDown64** [comment]
Also how come we changed back from "fpga" to "ice40" and got rid of the "tb" folder?

**Review by ajprende — APPROVED** [review]
Looks all set.

**Review by sbowles22 — DISMISSED** [review]
lgtm B)

**Comment by ajprende** [comment]
> @lagudu-glitch Can we restore "insert_project_name_here.sv" so that we can have both the FPGA top and ASIC top?

Shouldn't we be using the caravel wrapper specifically as the asic top? only we can run it anyway, not the students.

**Comment by StarsDown64** [comment]
> > @lagudu-glitch Can we restore "insert_project_name_here.sv" so that we can have both the FPGA top and ASIC top?
> 
> Shouldn't we be using the caravel wrapper specifically as the asic top? only we can run it anyway, not the students.

This is true, but the integration into caravel will have to be done through all those muxes and stuff, so it would be better to have a standardized project top so that we can consistently interface with them once we do caravel integration. Plus included the chip select pin will be good because if we don't a group may forget to implement it. We're not infallible so the reminder is good.

TLDR: It's not an ASIC top (caravel), it's a project top.

**Comment by StarsDown64** [comment]
TBF, we could also just enforce one project top and make them do the FPGA top through their project top. Thoughts?

**Review by sbowles22 — APPROVED** [review]
nvm changed my mind

**Review by sbowles22 — APPROVED** [review]
lgtm B)

**Review by StarsDown64 — APPROVED** [review]
